### ISSUE TITLE
E-mail servers status

### ISSUE THREAD

**Issue by gpotter2** [readthedocs/readthedocs.org#6030]
Hi & sorry for bothering.

Could you please confirm that the e-mailing servers are working properly? (or maybe there is a delay that I'm not aware of ? :smile:) I can't manage to get the verification email :/

I tried both a Gmail address and custom hosted one, to no avail :/ (I checked the spams & server logs. It doesn't look like SpamAssassin catched it, nor that it was ever received :/)

Thanks !

**Comment by stsewd** [comment]
Hi, sorry. The team in working into get email back again.

**Comment by gpotter2** [comment]
Thanks for the quick answer!

**Comment by davidfischer** [comment]
Just to give a bit more details, we've gotten something like ten thousand signups today (usually we get less than a hundred) from many different IPs and our system sends an email verification to each one. Unfortunately, after not many of these, our email sender has shut us off and is requiring us to talk to them. We are trying to get in touch and hoping to just have it come back without switching, but if we don't hear soon we're just going to swap providers at least temporarily.

**Comment by agjohnson** [comment]
Our current mail host, Mailgun, did not respond to us in a timely manner and we weren't any closer to resolution on the issue with them going into the weekend. I've moved our outbound mail to Sendgrid and mail should finally be sending again.

I tested a number of email notifications that we send, and they all seemed to go out okay. Let us know if any particular message doesn't seem to be sending still.
